src/server/migrations/migrateDatabase.js

```diff
@@ -0,0 +1,191 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const db = require('../models/database').db;
+const Migration = require('../models/Migration');
+const { compareSemanticVersion } = require('../util');
+const { log } = require('../log');
+
+// file needed to run database transaction
+const requiredFile = [];
```

Comment:
I don't like the idea of this being global state. It means we can only migrate once per run of the node server, and that's not good for testing.